docs/section11/Rule11-13.md

```diff
@@ -0,0 +1,34 @@
+# Service_Water_Heating - Rule 11-13  
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** FALSE  
+
+**Rule ID:** 11-13  
+
+**Rule Description:** "Service water-heating energy consumption shall be calculated explicitly based upon the volume of service water heating required and the entering makeup water and the leaving service water-heating temperatures. Entering water temperatures shall be estimated based upon the location. Leaving temperatures shall be based upon the end-use  requirements."  
+
+**Rule Assertion:** UNDETERMINED / NOT_APPLICABLE  
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, (e)  
+
+**Evaluation Context:** B_RMD - each building  
+**Data Lookup:**   
+**Function Call:**  
+- NONE  
+
+**Applicability Checks:**  
+- check that there is SWH in the proposed model AND that the building has a 24hr operating schedule
+
+## Applicability Checks:  
+- look at each building in the RMD: `for building in B_RMD:`
+    - create a boolean is_applicable and set it to false: `is_applicable = false`
+    - look at each space in the building: `for space in P_RMD.building...:`
+        - check to see if there is at least one SWH use associated with the space: `if len(space.service_water_heating_uses) > 0:`
+            - the rule is applicable: `is_applicable = true`
+      
+    
+    ## Rule Assertion:
+    - if the boolean is_applicable is equal to true, the rule is applicable, return UNDETERMINED and a note letting the reviewer know what to check: `if is_applicable: UNDETERMINED raise_message: "This building has service water heating loads.  Confirm that service water heating energy consumption is calculated explicitly based upon the volume of service water heating required and the entering makeup water and leaving service water heating temperatures.  Entering water temperatures shall be estimated based upon the location. Leaving temperatures shall be based upon the end-use  requirements."`
```

Comment:
I think we should discuss this on the call. Seems this requires the reviewer to dig into how the simulation tool calculates SWH consumption which seems like a big ask.

Reply:
Agreed, and it seems likely that the simulation tool would be consistent in how it calculates SWH consumption. Would it make sense to add a data element like calculation_method with options?

Just thinking out loud - What are alternative ways that it might be calculated and result in a FAIL? If it considers change in density?

Reply:
Agreed that it's a big ask, but not sure how to achieve it given the information we currently have.  We can give the reviewer more information here, such as the entering and leaving temperatures and the calculated energy required to heat the water, but without weather data and a simulation we can't double check the energy used in the model.  Thoughts?

Reply:
Brainstorming, what about outputting a Load/hr-sf and Load/sf-year calculated using the approach used in the function you created Karen?

Reply:
I created a dict `service_water_heating_info` and included btu/year and btu/sf/year - btu/hr is a bit complicated in the case where there are multiple `ServiceWaterHeatingUses`.  For now I didn't provide this information in the message, the intention is that the dict is available in calc_vals.  Thoughts on delivery?